apps/common/test/lexical/line_parser_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule Lexical.SourceFile.LineParserTest do
+  alias Lexical.SourceFile.LineParser
+
+  import Lexical.SourceFile.Line
+  use ExUnit.Case
+  use ExUnitProperties
+
+  test "parsing a single \n" do
+    assert length(LineParser.parse("\n", 1)) == 1
+  end
+
+  def parse(lines) do
```

Comment:
I noticed that you often use 'public def' instead of 'defp' in tests. I'm curious about the reason behind this choice.

Reply:
I don't give it much thought tbh. They're unit tests, they're supposed to be simple and dumb so I use def. 
One thing to consider is that setup and callback functions need to be def.